### ISSUE TITLE
obs-ndi cant find libndi

### ISSUE THREAD

**Issue by qwarlock** [Palakis/obs-ndi#566]
I am trying to install obs-ndi on my slackware system.  I have tried the 4.9 4.8 4.7 binaries with their accompanying libndi.  obs boots and pops a box saying it can not find libndi.  But, obs output says it finds libndi and shows the location and version.  So, obs finds the plugin. Obs finds the libndi.  But, the plugin can not find the library I think.  Any help with this?  It would be cool to use this software but so far can not get it to connect up.  Thanks in advance.

**Comment by qwarlock** [comment]
info: [obs-ndi] hello ! (version 4.7.1)
info: [obs-ndi] Trying ''
info: [obs-ndi] Trying '/usr/lib'
info: [obs-ndi] Found NDI library at '/usr/lib/libndi.so.4'
info: [obs-ndi] Trying '/usr/local/lib'
info: [obs-ndi] Found NDI library at '/usr/local/lib/libndi.so.4'
error: [obs-ndi] Can't find the NDI library

**Comment by qwarlock** [comment]
nevermind.   I converted your libndi  deb package to slackware package and installed it.  I then did ldd ndi-record and saw that it could not find the avahi libraries.  They are in /usr/lib64/lib directory.  I soft linked it over to /usr/lib64 and then ran ldd again.  It found everything fine.  Then I booted up OBS and there they are... NDI source and cam.... So this is fixed.  Is the source code for libndi floating around anywhere?  It would have been easier to just compile and install it.  Anyway looks very cool!
